osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -474,6 +474,7 @@ private void modButtonPressed(Mod selectedMod)
             {
                 if (State.Value == Visibility.Visible) sampleOn?.Play();
                 DeselectTypes(selectedMod.IncompatibleMods, true);
+                if (selectedMod.RequiresConfiguration) ModSettingsContainer.Alpha = 1;
```

Comment:
From a code readability prespective, since there is more than one `if` statement here, would be better to add empty line between each line.

Reply:
Resolved, I guess?